<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>newphoto</class>
 <widget class="QDialog" name="newphoto">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>753</width>
    <height>593</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Nueva Foto</string>
  </property>
  <widget class="QPushButton" name="ImagenBoton">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>550</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>Imagen</string>
   </property>
  </widget>
  <widget class="QPushButton" name="doneBoton">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>550</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>AK7</string>
   </property>
  </widget>
  <widget class="QLabel" name="fotolabel">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>140</y>
     <width>341</width>
     <height>291</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>70</y>
     <width>160</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QLabel" name="nombre">
      <property name="text">
       <string>Nombre</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="ano">
      <property name="text">
       <string>Ano</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="autor">
      <property name="text">
       <string>Autor</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="tamano">
      <property name="text">
       <string>Tamano</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="descripcion">
      <property name="text">
       <string>Descripcion</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="galeria">
      <property name="text">
       <string>Galeria</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>50</y>
     <width>160</width>
     <height>501</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLineEdit" name="nombreText"/>
    </item>
    <item>
     <widget class="QLineEdit" name="anoText"/>
    </item>
    <item>
     <widget class="QLineEdit" name="autorText"/>
    </item>
    <item>
     <widget class="QLineEdit" name="tamanoText"/>
    </item>
    <item>
     <widget class="QLineEdit" name="descripcionText"/>
    </item>
    <item>
     <widget class="QLineEdit" name="galeriaText"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
